Makefile

```
@@ -698,3 +698,52 @@ endif
 
 run_lib_files_test:
 	docker run --entrypoint bash -v $(realpath tests/file_lists):/test $(OVMS_CPP_DOCKER_IMAGE):$(OVMS_CPP_IMAGE_TAG)$(IMAGE_TAG_SUFFIX) ./test/test_release_files.sh ${BAZEL_DEBUG_FLAGS} > file_test.log 2>&1 ; exit_status=$$? ; tail -200 file_test.log ; exit $$exit_status
+
+# Multi-arch build targets
+.PHONY: docker_build_multiarch
+docker_build_multiarch:
```

checkmake (0.3.2)

[warning] Target body for "docker_build_multiarch" exceeds allowed length of 5 lines (9).

(maxbodylength)